ts/frames.ts

```diff
@@ -1369,7 +1369,7 @@ export const frameDefinitions: Record<FrameType, FrameDefinition> = {
 
 };
 
-type FrameStateMetadata = {
+export type FrameStateMetadata = {
```

Comment:
This export is never imported.

```suggestion
type FrameStateMetadata = {
```